src/spotify_to_tidal/sync.py

```diff
@@ -231,7 +278,7 @@ def sync_playlist(spotify_session: spotipy.Spotify, tidal_session: tidalapi.Sess
     for index, tidal_track in enumerate(tidal_tracks):
```

Comment:
`tidal_track_id`

Reply:
Actually I'm confused now, does `tidal_search` return a full track object when there wasn't a cache match, but only a track id when there was? 😕 Is this intentional?